.gitignore

```diff
@@ -4,6 +4,11 @@
 .env
 .DS_Store
 venv/
+.venv/
 .pytest_cache/
 **__pycache__/
 db.sqlite3
+*.pyc
+.vs/
+.git/
+**__init__.py
```

Comment:
There is no need to exclude `__init__.py` files, they could be present in the project